migrations/initial_schema.sql

```diff
@@ -3,7 +3,7 @@
 --
 
 -- Dumped from database version 14.5 (Debian 14.5-2.pgdg110+2)
--- Dumped by pg_dump version 14.5
+-- Dumped by pg_dump version 14.5 (Ubuntu 14.5-0ubuntu0.22.04.1)
```

Comment:
Ignore what I wrote in the README about updating this file. It only needs to represent the state of the DB pre-migrations and will not be updated. 

When I get around to it, we will start tracking the "current" schema in a file, which we will produce at the end of migrations such that anyone can restore the current schema directly (as opposed to applying the initial SQL and then all migrations).

TLDR; let's revert changes to this file for now.

Reply:
Ahh sorry I think I must have misunderstood - I get you now

Reply:
Nope, pretty sure it was just me doing a full 180 in the docs. :sweat_smile: